examples/expo-react-native/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "expo-react-native",
+  "main": "expo-router/entry",
+  "version": "1.0.0",
+  "scripts": {
```

Comment:
do we want to add a script that removes the `node_modules/react & node_modules/react-dom` that can run automatically?

Reply:
I thought I solved that problem with the linked react dependencies below. Doesn't it work on your machine?

Reply:
I think this part may work, didn't seem to have to do it.